README.md

```diff
@@ -0,0 +1,16 @@
+# Welcome to Quarkiverse!
+
+The Quarkiverse GitHub organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+
+You can browse the [Quarkiverse documentation](./docs/Home.md) to get more info.
+
+## Here is the flow to get your extension onboarded:
+
+1. The first step is to create an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) (make sure to check first if the extension you’re proposing is not created yet) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss if the proposal extension should belong to core or live in the Quarkiverse Hub.
```

Comment:
```suggestion
1. The first step is to create or pick an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss if the proposal extension should belong to core or live in the Quarkiverse Hub.
```

Reply:
maybe ask to mention @quarkiverse-team ? (and create that team)

Reply:
@maxandersen we could automate that right?

Reply:
Not sure what hint we would use to know when to add the cc  hence asking user to do it.